sc/core/Crone.sc

```diff
@@ -81,6 +81,8 @@ Crone {
 		//Crone.runShellCommand("jack_connect \"crone:output_6\" \"supernova:input_2\"");
 		Crone.runShellCommand("jack_connect \"crone:output_5\" \"SuperCollider:in_1\"");
 		Crone.runShellCommand("jack_connect \"crone:output_6\" \"SuperCollider:in_2\"");
+		Crone.runShellCommand("jack_connect softcut:output_1 SuperCollider:in_1");
```

Comment:
this will create an always-on feedback path from softcut to supercollider, which is undesirable in many situations.

i would at least create an additional, separate input bus pair when starting the scsynth process. but i'm not sure i see this as a big win for functionality.

Reply:
sorry about that! fixed.